### PR TITLE
Cyren ThreatInDepth Base Content Pack v1.1.0

### PR DESCRIPTION
**Pull request by meilke**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N/A

## Description

* new indicator fields (feed relationships, IP intensity/risk, source tags)
* related indicators (for instance a SHA256 that has been hosted by a malicious URL) will now be
  * imported as indicator as well (using a Cyren Source Tags field with value `related` to differentiate between a full indicator)
  * featured in a field called Cyren Feed Relationships (basically a beefed up version of Feed Related Indicators) - bidirectional
* accompanying script widget to show Cyren Feed Relationships
* indicator layout template to showcase Cyren Threat InDepth custom fields

## Screenshots

image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [x] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        "Timestamp UTC": timestamp_human_readable,
+        "Entity Category": entity_category,
+    }
+
+    return indicator_object
+
+
+def cyren_feed_relationship(args) -> CommandResults:
+    indicator = args["indicator"]
+    relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
```
Comment (abaumgarten) [inline Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py:39]: Is there a chance that `args` will not have `indicator` key? If there is such a chance, let's check it before calling `indicator.get` .
```suggestion
    indicator = args.get("indicator")
    if indicator:
        relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
   else:
        `choose what you would like to do in this case`
```

**Review by abaumgarten — COMMENTED** [review]
@meilke, Thank you for your contribution! Seems like you did a wonderful job!
Please see my only one comment below. 
Your PR includes many Content elements (such as indicator_fileds, layout, widget, etc ) that should be tested during a demo session. After the demo session will might be some more fixes from your side.
Feel free to send me a dm on Slack if something is not clear.
Thanks.

File: Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py (hunk 1 above)
Comment (meilke) [reply Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py:39]: It is probably a good idea. Although this is a script with a tag of `dynamic-indicator-widget` and I suspect the software to guarantee the property.

**Comment by abaumgarten** [comment]
Hi, one more thing please, because you are a Partner, you should add your pack a publisher logo. This logo will be displayed on the pack details view (in MarketPlace). 
In order to add the logo, you should edit the `pack_metadata.json` file: add a link to a publisher logo under a key called `authorImage`.
More information about the logo you can find here: https://xsoar.pan.dev/docs/integrations/packs-format#author_imagepng.
Thank you.

**Comment by meilke** [comment]
> Hi, one more thing please, because you are a Partner, you should add your pack a publisher logo. This logo will be displayed on the pack details view (in MarketPlace).
> In order to add the logo, you should edit the `pack_metadata.json` file: add a link to a publisher logo under a key called `authorImage`.
> More information about the logo you can find here: https://xsoar.pan.dev/docs/integrations/packs-format#author_imagepng.
> Thank you.

done

File: Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py (hunk 1 above)
Comment (meilke) [reply Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py:42]: done

**Comment by abaumgarten** [comment]
@meilke we are all set, approving the PR
Thanks!

**Review by abaumgarten — APPROVED** [review]
Great job, thank you!

**Pull request merged**